docs/source/examples/mg-mode-part-i.ipynb

```diff
@@ -0,0 +1,926 @@
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# Create Geometry and set root Universe\n",
+    "geometry = openmc.Geometry()\n",
+    "geometry.root_universe = root_universe\n",
+    "\n",
+    "# Export to \"geometry.xml\"\n",
+    "geometry.export_to_xml()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can now create the tally file information. The tallies will be set up to  give us the pin powers in this notebook. We will do this with a mesh filter, with one mesh cell per pin."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "tallies_file = openmc.Tallies()\n",
+    "\n",
+    "# Instantiate a tally Mesh\n",
+    "mesh = openmc.Mesh()\n",
+    "mesh.type = 'regular'\n",
+    "mesh.dimension = [17 * 2, 17 * 2]\n",
+    "mesh.lower_left = [-32.13, -10.71]\n",
+    "mesh.upper_right = [+10.71, +32.13]\n",
+    "\n",
+    "# Instantiate tally Filter\n",
+    "mesh_filter = openmc.MeshFilter(mesh)\n",
+    "\n",
+    "# Instantiate the Tally\n",
+    "tally = openmc.Tally(name='mesh tally')\n",
+    "tally.filters = [mesh_filter]\n",
+    "tally.scores = ['fission']\n",
+    "\n",
+    "# Add tally to collection\n",
+    "tallies_file.append(tally)\n",
+    "\n",
+    "# Export all tallies to a \"tallies.xml\" file\n",
+    "tallies_file.export_to_xml()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With the geometry and materials finished, we now just need to define simulation parameters for the `settings.xml` file. Note the use of the `energy_mode` attribute of our `settings_file` object. This is used to tell OpenMC that we intend to run in multi-group mode instead of the default continuous-energy mode. If we didn't specify this but our cross sections file was not a continuous-energy data set, then OpenMC would complain."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# OpenMC simulation parameters\n",
+    "batches = 1000\n",
+    "inactive = 100\n",
+    "particles = 5000\n",
+    "\n",
+    "# Instantiate a Settings object\n",
+    "settings_file = openmc.Settings()\n",
+    "settings_file.batches = batches\n",
+    "settings_file.inactive = inactive\n",
+    "settings_file.particles = particles\n",
+    "\n",
+    "# We dont need a tallies.out file, as we will be working with the StatePoint\n",
+    "# file instead, so tell OpenMC not to write the file.\n",
+    "settings_file.output = {'tallies': False}\n",
+    "\n",
+    "# Tell OpenMC this is a multi-group problem\n",
+    "settings_file.energy_mode = 'multi-group'\n",
+    "\n",
+    "# Set the verbosity to 6 so we dont see output for every batch\n",
+    "settings_file.verbosity = 6\n",
```

Comment:
I took the view here that since this is not the first of many examples, that the user has already seen the iterations and will just be scrolling past anyways.  Displaying progress is only for us, the ones making the notebook, not for those learning from it.  Since I'd rather run more particles to get attractive results but not want to require 30 pages of printouts, this was the compromise.  Thoughts?